### ISSUE TITLE
pull data from periodictable project

### ISSUE THREAD

**Issue by tacaswell** [XraySpectroscopy/XrayDB#6]
http://www.reflectometry.org/danse/elements.html

https://pypi.python.org/pypi/periodictable

This seems to have:
- natural crystal structures
- density
- constants 
- stable ions
- covalent radius 
- Cromer-Mann (which interestingly in copyright Columbia via @pavoljuhas and @sbillinge)
- magnetic form-factor
- mass / abundance for isotopes
- neutron scattering factors
- xray-scattering factors from Henke et al

**Comment by pavoljuhas** [comment]
There is a conda recipe for periodictable at
https://github.com/diffpy/conda-recipes/tree/master/periodictable
Any option to just use it as a dependency?

**Comment by sbillinge** [comment]
is it still be supported/extended by the reflectometry group?

Before pulling in we may want to discuss with them if they think it is a
good idea and if scikit-beam is the best "ultimate resting place (URP)" for
such an important community resource.

If scikit-beam becomes what we hope it becomes (like scipy say) it just
might be.

S

On Fri, Apr 8, 2016 at 1:13 AM, Pavol Juhas notifications@github.com
wrote:

> There is a conda recipe for periodictable at
> https://github.com/diffpy/conda-recipes/tree/master/periodictable
> Any option to just use it as a dependency?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/scikit-beam/XrayDB/issues/6#issuecomment-207217960

## 

Prof. Simon Billinge
Applied Physics & Applied Mathematics
Columbia University
500 West 120th Street
Room 200 Mudd, MC 4701
New York, NY 10027
Tel: (212)-854-2918 (o) 851-7428 (lab)

Condensed Matter Physics and Materials Science Dept.
Brookhaven National Laboratory
P.O. Box 5000
Upton, NY 11973-5000
(631)-344-5661

email: sb2896 at columbia dot edu
home: http:// http://nirt.pa.msu.edu/bgsite.apam.columbia.edu/

**Comment by newville** [comment]
Yes, those look very useful.   XrayDB does have a few of the X-ray versions of these resources already:
-   The Waasmaier and Kirfel data for f0(q) should replace Cromer-Mann, I believe, but having both is OK too.
-   The Elam et al and Chantler tables have X-ray attenuation and scattering cross-sections that should be better than Henke et al  (which do not go above 30 keV).   The EADL / EPDL data also contains similar data.  But, similarly, including the Henke data is OK with me.

Anyway, I'm +1 on adding any data resources and seeing what gets used.  I wouldn't want to insist this is the only place this data is, but putting it all it one place with a uniform-ish would be a big step forward.   If it includes neutron data, perhaps the name "xraydb" is too limiting?  

FWIW, there might also be interesting code and resources for sckit-beam from the http://pythonhosted.org//xrt/ project.  This has a lot of ray-tracing like functionality, but also includes X-ray source calculations and dynamical diffraction capabilities.
